README.md

```diff
@@ -153,9 +153,88 @@ This library doesn't handle this for you, but here are the 2 main options:
 This is the easiest method: by rendering your form inside a live component,
 it will automatically re-render when the form changes.
 
-### B) Write custom JavaScript
+### B) Use [Symfony UX Turbo](https://symfony.com/bundles/ux-turbo/current/index.html#decomposing-complex-pages-with-turbo-frames)
+
+If you are already using Symfony UX Turbo on your website, you can have a dynamic form running quickly without any JavaScript.
+
+Or you may want to install Symfony UX Turbo, [check out the documentation](https://symfony.com/bundles/ux-turbo/current/index.html#installation).
+
+> [!NOTE]
+> You only need to have Turbo Frame, you can disable Turbo Drive if you do not use it, or do not want to use it.
+> ie: `Turbo.session.drive = false;`
+
+Simply add a `<turbo-frame>` around your form:
+
+```twig
+<turbo-frame id="rating-form">
+{{ form(form) }}
+</turbo-frame>
+```
+
+From here you need two small changes:
+
+First, in your form type:
```

Comment:
let's describe with words what exactly you need to do in your form type... it's not pretty clear from the code IMO.

Also, I think we can use GitHub's `diff` code block to show what lines of code were added. Wdyt?